src/components/Header/index.js

```diff
@@ -3,16 +3,81 @@ import Link from "gatsby-link";
 
 import "./index.scss";
 
-const Header = () => (
-  <div className="Header">
-    <div className="Header-inner">
-      <h1 style={{ margin: 0 }}>
-        <Link to="/" className="Header-link">
-          Include Braga
-        </Link>
-      </h1>
-    </div>
-  </div>
-);
-
-export default Header;
+const sections = [
+  {
+    name: "Section 1",
+    link: "#section1",
+  },
+  {
+    name: "Section 2",
+    link: "#section2",
+  },
+  {
+    name: "Section 3",
+    link: "#section3",
+  },
+  {
+    name: "Section 4",
+    link: "#section4",
+  },
+];
```

Comment:
How do you feel about putting this in a `data.js` file and importing it from there?

Reply:
This is not going to get reused atm, so I think it's not needed right now. Better to keep things tidy.